server/services/project_service.py

```diff
@@ -77,16 +77,24 @@ def get_contribs_by_day(project_id: int) -> ProjectContribsDTO:
 
         contribs_dto = ProjectContribsDTO()
         dates = list(set(r[1] for r in stats))
-        dates.sort(reverse=True)
+        dates.sort(reverse=False)  # Why was this reversed?
```

Comment:
@JorgeMartinezG Not sure why this was reversed, hopefully nothing unexpected broke when I un-reversed it for the cumulative counts.
https://github.com/hotosm/tasking-manager/blame/5c151ba8cd60aac72a194fb4e67b2746cdc49d34/server/services/project_service.py#L80

Reply:
If you can suggest a way to get the total number of tasks after the splits from the backend, I think that would help the visualization a lot! Right now 300% is hard to understand.

Reply:
@thadk the reversed=True is for listing the most recent dates first. cc @willemarcel

Reply:
@JorgeMartinezG Thanks! Do you see how in the screenshot visualization "Contributions Timeline" the percent's are above 100% – do you know a way to get the number of tasks after splitting on the backend?

Reply:
you mean the total_tasks column is not giving the correct value? One quick way is to filter all tasks given project id, like:
```
Task.query.with_entities(func.count(Task.project_id).label('total'))
.filter(Task.project_id==project_id)
.group_by(Task.project_id)
```

Have not tested locally, but it should work

Reply:
Just saw this: I will try that @JorgeMartinezG

Reply:
@JorgeMartinezG No, that is returning the same number as `project.total_tasks`. At least for https://xenodochial-ramanujan-f0fd23.netlify.com/projects/6175  it should be greater than `1150`.

Reply:
Oh, I counted the number of tasks in the geojson and it was also 1150, even though the highest ID is 1168. I wonder if the backend code is getting the cumulative number of task editing sessions opened instead of cumulative number of tasks mapped or validated. Can someone take this backend back over, please?